browser/src/dom/NotebookbarAccessibility.js

```diff
@@ -0,0 +1,288 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+	This class is used for managing the accessibility keys of notebookbar control.
+*/
+
+/* global app NotebookbarAccessibilityDefinitions */
+
+var NotebookbarAccessibility = function() {
+
+	this.initialized = false;
+
+	this.activeTabPointers = {
+		id: '',
+		contentList: [],
+		infoBoxList: []
+	},
+
+	this.definitions = new NotebookbarAccessibilityDefinitions();
+	this.tabInfoList = null; // This is to be fetched from NotebookbarAccessibilityDefinitions class at initialization.
+	this.combination = null;
+	this.filteredItem = null;
+
+	this.addInfoBox = function(acnhorElement) {
+		var infoBox = document.createElement('div');
+		infoBox.classList.add('accessibility-info-box');
+		infoBox.textContent = acnhorElement.accessKey;
+		var rectangle = acnhorElement.getBoundingClientRect();
+		infoBox.style.top = (rectangle.top + 20) + 'px';
+		infoBox.style.left = rectangle.left + 'px';
+		document.body.appendChild(infoBox);
+		return infoBox;
+	};
+
+	this.removeCurrentTabInfoBoxesAndAcceleratorKeys = function() {
+		for (var i = 0; i < this.activeTabPointers.contentList.length; i++) {
+			document.getElementById(this.activeTabPointers.contentList[i].id).accessKey = null;
```

Comment:
what if we will modify id's in the future in the notebookbar but not here in definitions?
I would recommend to first try to get element by id, then check if it exists. Also when it doesn't exist: console.error - so we will have info in the console that something was changed